### ISSUE TITLE
auto rotation problem

### ISSUE THREAD

**Issue by syddiqua** [verot/class.upload.php#142]
Around line 3300 where exif jpeg rotation occurs, i tested it with simple mobile phone images, and my server dropped a timeout (30s). Can you change the pixel by pixel method to php imagerotate function? Thats much faster. Thanks

**Comment by verot** [comment]
Yes, it would be faster. I don't have time to do it now, but I will keep this issue open and do it when I can.
